src/deltacode/test_utils.py

```diff
@@ -168,3 +169,14 @@ def streamline_headers(headers):
     headers.pop('deltacode_version', None)
     headers.pop('deltacode_options', None)
     streamline_errors(headers['deltacode_errors'])
+
+
+def fetch_files(location):
+    codebase = VirtualCodebase(location)
```

Comment:
This function has been extensively added to `test_utils` so as to prevent the creation of `deltacode` object for the test cases when only the `Resource files` are required